alibi_detect/cd/lsdd.py

```diff
@@ -76,6 +77,9 @@ def __init__(
         """
         super().__init__()
```

Comment:
Not part of this PR, but noticed that these `super` calls don't seem to do anything as there is no superclass constructor. This is probably a relic of when we inherited the public classes from `BaseDetector` - conscious that we should revisit this again to enforce expectations on the class hierarchy - but don't remember if there's an issue in place?

Reply:
Yeh I can't recall. I did remove once as a test and think it was fine, but should try again. Also added to this PR: https://github.com/SeldonIO/alibi-detect/issues/512.